index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Getting started overview
-summary: "GraphWalker is a open source Model-based testing tool for test automation. It's designed to make it easy to design your tests using graphs."
+summary: "GraphWalker is an open source Model-based testing tool for test automation. 
```

Comment:
I have transferred some sentences on new lines due to some lines are very long and it is not readable in IDE.
It requires horizontal scrolling and anybody who read it may lost focus on which line he was.